addons/Group_Chat_Button/src/style.css

```diff
@@ -0,0 +1,28 @@
+.group-chat-button-wrapper {
+    border: 0 solid black;
+    padding: 0;
+    margin: 0;
+}
+
+.group-chat-button-element {
+    border: 0 solid black;
+    padding: 0;
+    margin: 0;
+    background: transparent;
```

Comment:
Z tego co się orientuje to jeden background od razu zostanie nadpisany przez drugi. Sprawdzałem w kodzie icplayera i widzę, że ktoś podobnie robił ale nie wiem dlaczego... Czy ma to zastosowanie ?

Reply:
Szczerze mówiąc słabo znam się na CSS i skopiowałem to właśnie z innego źródła. Być może gdy nie będzie tła pod podanym adresem url, to wtedy tło się nie nadpisuje? Ewentualnie ktoś się pomylił i mogę to poprawić 😉